docs/user/user/pings/data-control-plane/index.md

```diff
@@ -0,0 +1,20 @@
+# Data Control Plane (a.k.a. Server Knobs)
```

Comment:
Probably worth cross-linking to and from `docs/user/user/metrics/data-control-plane`, huh